WorkInProgress/Administration/xASL_adm_CoregisterAtlas.m

```diff
@@ -1,4 +1,4 @@
-% Copyright 2015-2024 ExploreASL (Works In Progress code)
+% Copyright 2015-2025 ExploreASL (Works In Progress code)
```

Comment:
Please change the name to xASL_adm_RegisterAtlas or something, coregistration has the same meaning, and if it differs, coregistration is used for linear/intra-subject registration.

Reply:
Yes. Agree. Will do...